### DISCUSSION TITLE
Is it possible to sort docs descend or ascend by date？

### DISCUSSION THREAD

**Discussion by abcpeak** [facebook/docusaurus#8896] [Q&A] [answered]
I know the blog plug has `sortPosts` string to use:
'descending' |'ascending'. But I didn't find docs has it.

If docs file names are: 2023-01-01-Ana.md, 2023-01-02-Aaron.md, 2023-01-03-Baron.md....

Now the up-to-date file always display at the bottom. But I want it display at the top. 

So how to do it? descending or ascending?

**Comment by slorber** [comment, marked as answer]
If you are using explicit sidebars, you define the order yourself.

If you are using auto-generated sidebars, you can tweak the sidebar generator logic to implement your own ordering convention: https://docusaurus.io/docs/sidebar/autogenerated#customize-the-sidebar-items-generator

<img width="983" alt="CleanShot 2023-04-19 at 16 38 27@2x" src="https://user-images.githubusercontent.com/749374/233110384-cbad6474-1275-461d-af22-108d8333a93f.png">

**Reply by abcpeak** [reply]
I'm using auto-generated sidebar, How to change the logic from ascending (default) to descending?

**Reply by slorber** [reply]
Solution described above, using `sidebarItems.sort(yourComparator)` instead of `sidebarItems.reverse()`
